CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## X.X.X (YYYY-MM-DD)
 
+-   Go To Definition now works for class names too.
```

Comment:
Was about to suggest using the UI macros, then remembered this was not Asciidoc. 😭
Still, some kind of emphasis on the feature name would make this feel more natural.

```suggestion
-   _Go To Definition_ now works for class names too.
```